Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+MAINTAINER Shagun Khemka "shagun.khemka60@gmail.com"
+RUN apt-get update -y
+RUN apt-get install -y python-pip python-dev build-essential
+COPY . /app
+WORKDIR /app
+RUN pip install -r requirements.txt
```

Comment:
There is already a requirements.txt file in the app directory which would be /app/app/requirements.txt